WS2012R2/lisa/utilFunctions.ps1

```diff
@@ -253,6 +260,16 @@ function SetTestResult([String] $testName, [String] $completionCode)
             $newTestCase.failure.InnerText = "Test $testName Aborted."
         }
     }
+
+    $testID = GetTestID $testName $xmlData
+    foreach ($property in $newTestCase.properties.property)
+    {
+        if ($property.name -eq "TC_COVERED")
+        {
+            $property.value = $testID
```

Comment:
@chvalean Updated, if does not define TC_COVERED, output empty string to value, also change  TC_COVERED to lower case which looks better.
Thank you.